elastic/logs/challenges/esql-ccs-snapshot.json

```diff
@@ -0,0 +1,167 @@
+{% import "rally.helpers" as rally %}
+{
+  "name": "esql-ccs-snapshot",
+  "description": "Indexes logs to the local cluster, and runs ESQL queries either locally or across remote clusters via CCS. Data replicated via snapshot/restore.",
+  "schedule": [
+    {% if not p_skip_esql_ccs_remote_clusters %}
+      {% include "tasks/many-clusters-setup.json" %},
+    {% endif %}
+    {% include "tasks/index-setup.json" %},
+    {
+      "name": "bulk-index",
+      "operation": {
+        "operation-type": "raw-bulk",
+        "param-source": "processed-source",
+        "time-format": "milliseconds",
+        "profile": "fixed_interval",
+        "bulk-size": {{ p_bulk_size }},
+        "detailed-results": true
+      },
+      "clients": {{ p_bulk_indexing_clients }}{% if p_throttle_indexing %},
+      "ignore-response-error-level": "{{error_level | default('non-fatal')}}",
+      "schedule": "timestamp-throttler",
+      "max-delay-secs": 1
+      {% endif %}
+    },
+    {
+      "name": "force-merge-data-streams",
+      "operation": {
+        "operation-type": "force-merge",
+        "mode": "polling",
+        "max-num-segments": 1,
+        "request-timeout": 60
+      }
+    },
+    {% if p_skip_esql_ccs_remote_clusters %}
+      {
+        "name": "check-cluster-health-local",
+        "operation": {
+          "operation-type": "cluster-health",
+          "index": "_all",
+          "request-params": {
+            "wait_for_status": "{{cluster_health | default('green')}}",
+            "wait_for_no_relocating_shards": "true"
+          },
+          "retry-until-success": true
+        }
+      },
+    {% else %}
```

Comment:
Want to add a comment here, pointing to the corresponding `if`?

Reply:
Good idea, https://github.com/elastic/rally-tracks/pull/701/commits/13f190c2caa59c6c6c6c08868c40bfb790f7f69e.